web/styles/portico/landing_page.css

```diff
@@ -2284,6 +2285,30 @@ button {
         }
     }
 
+    .message-screenshot {
+        position: relative;
+
+        img {
+            display: block;
+            mask-image: linear-gradient(
+                to right,
+                hsl(0deg 0% 0%) 75%,
+                transparent 100%
+            );
+        }
+        margin-bottom: 10px;
+    }
+
+    .starred {
+        img {
+            mask-image: linear-gradient(
+                to right,
+                hsl(0deg 0% 0%) 95%,
+                transparent 100%
+            );
+        }
+    }
```

Comment:
What does this `starred` class mean? I feel like the name could be more clear/specific.

Reply:
`starred` is class used when screenshot has starred message --> reduces the fading effect of image so that the star is visible.

I updated it to `message-starred` to be more specific.

Reply:
OK, we should add a comment documenting this.

Reply:
added:
```    
/* message-starred is used when the message screenshot captured using
the script "generate-user-message-screenshot" contains starred message.
This is done to reduce the fading effect on the image caused by
".message-screenshot" class so that the star is visible. */
```